content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,209 @@
+
+But try to visualise as much as possible at every opportunity available, to be able to understand the changing nature of the data to hand.
+
+[Back to the index](#index)
+
+### Sanity Check
+
+You always want to be sure that your data are _exactly_ how you want them to be, and because of this is a good rule of thumb to apply a sanity check after the completion of this step.
+
+Although sometimes we won't know "what sanity checks to apply?" till we have run through the rest of the steps in the process and then come back the next iteration(s) and fine-tune this step so that it catches the important low-hanging fruits that do not hinder the rest of the processes. Sometimes you might just have to proceed with the data and the steps to follow i.e. Data Preparation, Data Cleaning, Data Visualisation, etc... will help reveal much about your collected data and decisions to take.
+
+As we talk about Data Visualisation, the Visualisation step in the previous section itelf is a good enough sanity check about your data - provided you have drawn those charts and have the skills to evaluate them. If not, hopefully the rest of the steps will help you capture those insights. And sometimes with experience and doing multiple iterations and observing various experiments visualisations on older datasets (snapshots) start to make sense.
+
+Then there is also ethics you do not want to miss out on and [the section to follow gives you insight into](#ethics). And also [interpretability/explainability](#interpretability--explainability) which overlap with each other.
+
+#### Ethics
+
+With rising concerns over privacy and bias, you definitely want to be sure that the data you can collected does respect the ethics and standards in the areas as much as possible.
+
+To help with that as the awareness about things are improving, there are a lot of resources available, one such place to start would be [here](https://github.com/neomatrix369/awesome-ai-ml-dl/blob/naster/README-details.md#ethics--altruistic-motives). One of the resources mentioned there is that of a python package called [Deon](https://pypi.org/project/deon/). Interestingly it has a digital checklist you can consult and see if they apply to what you are about to do.
+
+#### Interpretability / Explainability
+
+Another widely debated but essential topic is the ability for us to interpret or explain our models and what better place to start but during the data collection process. Another [good resource](https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/data/model-analysis-interpretation-explainability.md#post-model-creation-analysis-ml-interpretationexplainability
+) to look into when although it's an ever growing topic and subject to keep track of.
```

Comment:
Link to Google new page
https://www.tensorflow.org/resources/responsible-ai

here you find a lot of cool things, maybe something can fit here

like https://pair.withgoogle.com/guidebook/

Reply:
I think I have these links under the https://github.com/neomatrix369/awesome-ai-ml-dl/blob/master/data/model-analysis-interpretation-explainability.md#post-model-creation-analysis-ml-interpretationexplainability links but I'll share some again so its easier to find!

Reply:
Very useful links, I have looked at them and added them, if you think I missed any please let me know!